### DISCUSSION TITLE
All User OnBoarding Status

### DISCUSSION THREAD

**Discussion by ershadow786** [spatie/laravel-onboard#10] [Q&A]
Greetings,

Firstly thanks for such package.

Currently according to doc we can only able to fetch on boarding process status for current authenticated person only.
Is there any a way to get all users ```OnBoarding``` status, as this is one of feature required/process required as Admin.

Regards

**Comment by RhysLees** [comment]
You would just have to loop though your users, it dont say in the docs its limited to just the authenticated user:

```php
$users = User::all();

foreach($users as $user){
    $user->onboarding()->percentageCompleted();
}
```
